### ISSUE TITLE
It doesn't work for me

### ISSUE THREAD

**Issue by aminezazi** [sq1000000/Coinex-trading-bot#2]
I double checked every things but it doesn't work. just error
![Uploading Capture.PNG…]()

**Comment by sata1000000** [comment]
The image you attached didn't work. 
Are you sure you installed all the python dependencies and imported your API keys for both discord webhooks and the Coinex API?

**Comment by aminezazi** [comment]
tnx to responding . I checked dependencies again. seems ok.
API keys are ok too but keep getting error
![dep](https://github.com/sq1000000/Coinex-trading-bot/assets/104747959/ad76974c-5d90-48b7-94e9-da1022444da8)
![Capture](https://github.com/sq1000000/Coinex-trading-bot/assets/104747959/3584c741-4c40-4829-b836-c3aad0cd82a1)

**Comment by sata1000000** [comment]
Hmmm. Do you have enough money in your account for it to trade with? It's been years since I've last touched this project, so I forget how much exactly is needed, but I think you need at least ~$5 of both DOGE and USDT for the script to work properly

**Comment by aminezazi** [comment]
yes I do.. I think it my be related to coinex. I'm checking with them . I'll let you know the resault. tnks

**Comment by sata1000000** [comment]
Like I said, I haven't touched this script in a while. I just found out that Coinex has halted operations in the States. If you're in the US, that could be an issue.

https://www.wsj.com/articles/coinex-to-leave-u-s-after-settling-new-york-lawsuit-2e865279
